open-metadata-implementation/access-services/asset-lineage/asset-lineage-server/pom.xml

```diff
@@ -47,6 +47,18 @@
             <version>${junit.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.odpi.egeria</groupId>
```

Comment:
Only add the dependency once.

Reply:
Intellij somehow added both by itself - removed the duplicate